start.go

```diff
@@ -22,8 +22,8 @@ func startCmd() cli.Command {
 				Usage: "--log-level debug to enable debugging",
 			},
 			cli.BoolFlag{
-				Name:  "detach, d",
-				Usage: "Run container in background.",
+				Name:  "foreground, fg",
```

Comment:
I really prefer that stay as it is, it looks like the docker ones, so maybe adjust the logic to make fn run in the background, which is totally fine